### ISSUE TITLE
[FR] USE_CONTROLLER_FAN granular options

### ISSUE THREAD

**Issue by hamster65** [MarlinFirmware/Marlin#11253]
Just a thought / a nice to have:

USE_CONTROLLER_FAN activates the controller fan as soon as the heated bed starts to heat up. This is by intention to cool the mosfets.
My bed takes a long time to heat up and I usually sit nearby the printer to prepare the stl file. The fan noise is quite annoying. The mosfet has a heat sink and stays cool without any forced cooling. So it would be great to have more options here, like
USE_CONTROLLER_FAN_FOR_DRIVERS
USE_CONTROLLER_FAN_FOR_MOSFETS

**Comment by J-charles-C** [comment]
I,

For same effect, i have wired my driver's fan with my hotend fan. My bed heat up. When temp of bed is ok, my hotend begin to heat. At 50°, hotend and driver fan turn on.
Best

**Comment by boelle** [comment]
@hamster65 have you tried to use bigger fans? 80mm maybe?

**Comment by melslenstra** [comment]
Thanks @thisiskeithb for spotting that my FR #23401 is a duplicate of this one. Not sure how I managed to not find this before submitting my FR.

tl;dr: In my case, the controller fans are nowhere near the actual bed MOSFETs, so it makes even less sense to have them come on at full blast when I preheat my bed.
